camera_matrix:
  - [1008, 0.0, 780]
  - [0.0, 1007, 424]
  - [0.0, 0.0, 1.0]
dist_coeff: [-0.01955339, -0.00970571, -0.00145232,  0.00186774,  0.08397236]


#Camera matrix:
# [[1.00827669e+03 0.00000000e+00 7.79814266e+02]
# [0.00000000e+00 1.00679379e+03 4.24512922e+02]
# [0.00000000e+00 0.00000000e+00 1.00000000e+00]]
#Distortion coefficients:
# [[-0.01955339 -0.00970571 -0.00145232  0.00186774  0.08397236]]



# Max resolution
#camera_matrix:
#  - [1965.12, 0.0, 2304]
#  - [0.0, 1963.62, 1296]
#  - [0.0, 0.0, 1.0]
#dist_coeff: [-0.06776666,  0.15092009, -0.00069885,  0.00102074, -0.09718589]


